pkg/importer/writer_test.go

```diff
@@ -0,0 +1,30 @@
+package importer
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_writer_writeUnion(t *testing.T) {
+	tests := []struct {
+		name     string
+		arg      Type
+		expected string
+	}{
+		{"writeUnion",
+			&Union{name: "TestUnion", Options: FieldList{{Name: "Apple"}, {Name: "Orange"}}},
+			"!union TestUnion:\n    Apple\n    Orange\n",
+		},
+	}
+	for _, tt := range tests {
```

Comment:
Nit: Does this really need an array and a range?

Reply:
Definitely not necessary at the moment, but planning to add a bunch more test cases in a future PR 👍